### PR TITLE
Editing the help file for az consumption

### PR DESCRIPTION
**Pull request by sandeepnl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [x] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py [hunk 1]
```diff
@@ -7,15 +7,10 @@
 
 helps['consumption'] = """
     type: group
-    short-summary: Manage consumption of Azure resources.
-"""
-
-helps['consumption usage'] = """
-    type: group
-    short-summary: Inspect the usage of Azure resources.
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py]: Removing this just means there's now no help message for the `az consumption usage` group.
Either add this back in or change the command to `az consumption list-usage` as there's only a single command in that group.

File: src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py [hunk 2]
```diff
@@ -7,15 +7,10 @@
...
+    short-summary: View consumption of Azure resources.
 """
 
 helps['consumption usage list'] = """
     type: command
-    short-summary: Show the details of Azure resource consumption, either as an invoice or within a billing period.
+    short-summary: Shows the usage details of Azure resource consumption, either with a biling period or by a specific date range.
```
Comment (sptramer) [inline src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py]: Change to `by billing period or within a date range.`

File: src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py [hunk 3]
```diff
@@ -7,15 +7,10 @@
 
 helps['consumption'] = """
     type: group
-    short-summary: Manage consumption of Azure resources.
-"""
-
-helps['consumption usage'] = """
```
Comment (sptramer) [inline src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py]: Do not delete, or else the `usage` group will not show up when a user types `az consumption -h` which will hide commands. Instead provide a better short description.

File: src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py (hunk 1 above)
Comment (sandeepnl) [reply src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py]: I understand what you say. But when I look at the current documentation. It says "List of commands" and shows "az consumption usage" and "az consumption usage list". But there is only one command in reality and that is az consumption usage list
And to your comment to get rid of the group if we are not planning more commands under usage, yes we do plan to have more commands under usage coming soon.
So the question is, how to keep the group but not list as a command in the help documentation?

File: src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py (hunk 3 above)
Comment (sandeepnl) [reply src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py]: I understand what you say. But when I look at the current documentation. It says "List of commands" and shows "az consumption usage" and "az consumption usage list". But there is only one command in reality and that is az consumption usage list
And to your comment to get rid of the group if we are not planning more commands under usage, yes we do plan to have more commands under usage coming soon.
So the question is, how to keep the group but not list as a command in the help documentation?

File: src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py (hunk 1 above)
Comment (sptramer) [reply src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py]: @derekbekoe Is this a CLI bug regarding groups with only one command, or a feature?

**Comment by tjprescott** [comment]
Due to recent changes to the CLI core, you will need to rebase your changes against the latest in the dev branch.

**Comment by derekbekoe** [comment]
Closing due to inactivity.

**Pull request closed without merging**